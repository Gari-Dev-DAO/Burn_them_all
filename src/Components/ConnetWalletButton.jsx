import { WalletMultiButton} from "@solana/wallet-adapter-react-ui"

const ConnetWalletButton = () => {
  return (
    <div >
       <WalletMultiButton className="walletBtn"/>
    </div>
  )
}

export default ConnetWalletButton