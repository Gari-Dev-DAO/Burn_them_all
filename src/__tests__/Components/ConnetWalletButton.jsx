
import { render } from "@testing-library/react";
import ConnetWalletButton from "../../Components/ConnetWalletButton";


describe("Connect Wallet Button", () => {
  it("renders component", () => {
    const { container } = render(<ConnetWalletButton/>);
    expect(container).toMatchSnapshot();
  });
});
