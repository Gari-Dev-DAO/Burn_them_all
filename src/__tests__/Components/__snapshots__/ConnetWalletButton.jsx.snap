// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connect Wallet Button renders component 1`] = `
<div>
  <div>
    <button
      class="wallet-adapter-button walletBtn"
      tabindex="0"
      type="button"
    >
      Select Wallet
    </button>
  </div>
</div>
`;
